web/logixng_examples/logixng_sequence.js

```diff
@@ -34,375 +34,375 @@ var sensor2_Pos = 760;
-	}
-	if (sensorTriggered) {
-		// Sensor 2 triggered
-		if (true ||    !sensor2_active) {
-			sensor2_active = true;
-			jmri.setSensor("IS_7_2", 2);
-		}
-	} else {
-		// Sensor 2 not triggered
-		if (true ||    sensor2_active) {
-			sensor2_active = false;
-			jmri.setSensor("IS_7_2", 4);
-		}
-	}
-
-
-	sensorTriggered = false;
-	if (selectDivergedTrack) {
-		sensorTriggered = (((locoPos-locoLength/2) <= sensor3_Pos) && ((locoPos+locoLength/2) >= sensor3_Pos));
-		sensorTriggered |= (((carPos-carLength/2) <= sensor3_Pos) && ((carPos+carLength/2) >= sensor3_Pos));
-	}
-	if (sensorTriggered) {
-		// Sensor 3 triggered
-		if (true ||    !sensor3_active) {
-			sensor3_active = true;
-			jmri.setSensor("IS_7_3", 2);
-		}
-	} else {
-		// Sensor 3 not triggered
-		if (true ||    sensor3_active) {
-			sensor3_active = false;
-			jmri.setSensor("IS_7_3", 4);
-		}
-	}
+    var sensorTriggered = false;
+    sensorTriggered = (((locoPos-locoLength/2) <= sensor1_Pos) && ((locoPos+locoLength/2) >= sensor1_Pos));
+    sensorTriggered |= (((carPos-carLength/2) <= sensor1_Pos) && ((carPos+carLength/2) >= sensor1_Pos));
+
+    if (sensorTriggered) {
+        // Sensor 1 triggered
+        if (true ||    !sensor1_active) {
+            sensor1_active = true;
+            jmri.setSensor("IS_7_1", 2);
+        }
+    } else {
+        // Sensor 1 not triggered
+        if (true ||    sensor1_active) {
+            sensor1_active = false;
+            jmri.setSensor("IS_7_1", 4);
+        }
+    }
+
+
+    sensorTriggered = false;
+    if (!selectDivergedTrack) {
+        sensorTriggered = (((locoPos-locoLength/2) <= sensor2_Pos) && ((locoPos+locoLength/2) >= sensor2_Pos));
+        sensorTriggered |= (((carPos-carLength/2) <= sensor2_Pos) && ((carPos+carLength/2) >= sensor2_Pos));
+    }
+    if (sensorTriggered) {
+        // Sensor 2 triggered
+        if (true ||    !sensor2_active) {
+            sensor2_active = true;
+            jmri.setSensor("IS_7_2", 2);
+        }
+    } else {
+        // Sensor 2 not triggered
+        if (true ||    sensor2_active) {
+            sensor2_active = false;
+            jmri.setSensor("IS_7_2", 4);
+        }
+    }
+
+
+    sensorTriggered = false;
+    if (selectDivergedTrack) {
+        sensorTriggered = (((locoPos-locoLength/2) <= sensor3_Pos) && ((locoPos+locoLength/2) >= sensor3_Pos));
+        sensorTriggered |= (((carPos-carLength/2) <= sensor3_Pos) && ((carPos+carLength/2) >= sensor3_Pos));
+    }
+    if (sensorTriggered) {
+        // Sensor 3 triggered
+        if (true ||    !sensor3_active) {
+            sensor3_active = true;
+            jmri.setSensor("IS_7_3", 2);
+        }
+    } else {
+        // Sensor 3 not triggered
+        if (true ||    sensor3_active) {
+            sensor3_active = false;
+            jmri.setSensor("IS_7_3", 4);
+        }
+    }
 }
 
 
 
 
 
-window.setInterval(runTrain, 50);
+// window.setInterval(runTrain, 50);
```

Comment:
Why is this line commented? It seems to be a functional change.

Tip: I found this line by selecting the option `Hide whitespace` on GitHub when viewing this PR.

Reply:
Apologies for the unintended change @danielb987 , I was having a quick play with the file.
1st time I've looked at the WhiteSpace option in Github, good to be aware of it, thanks.